src/global/constants.ts

```diff
@@ -16,6 +16,12 @@ export const constants = {
     },
     userIds: {
       alfav: '259419275457069056',
+      Miroito: '240393399386112000',
+    },
+    emojis: {
+      arrow_left: '\u2b05',
+      arrow_right: '\u27a1',
+      white_check_mark: '\u2705',
```

Comment:
Field names should be in camelCase.

Reply:
The names on discord use this convention, should I still change it to camelCase and then we eventually use a library to convert it back to their original names?

Reply:
Yes, you never pass these names to discord.js so use camelCase for our code.

**Also, the strings should be `'\u2b05\ufe0f'` and `'\u27a1\ufe0f'`.**

Reply:
Names still not camelCase.